### ISSUE TITLE
Generate Fabric Mod don't work

### ISSUE THREAD

**Issue by Rdna123** [DemonWav/MinecraftDevIntelliJ#979]
The generated fabric mods do not work because of duplicate files caused by:

```

processResources {
    inputs.property "version", project.version

    from(sourceSets.main.resources.srcDirs) {
        include "fabric.mod.json"
        expand "version": project.version
    }

    from(sourceSets.main.resources.srcDirs) {
        exclude "fabric.mod.json"
    }
}
```

**Comment by RedNesto** [comment]
I _think_ the `from` closures are messing things up, removing them and keeping the `include` and `expand` statements at top level fixes it for me.

**Comment by Tom-The-Geek** [comment]
The official example mod was updated to fix this issue, while keeping the replacement only in the `fabric.mod.json`, so you could look there

**Comment by RedNesto** [comment]
Oh right: https://www.github.com/FabricMC/fabric-example-mod/tree/1.17/build.gradle

I'll fix this issue with #929
